examples/azure-cni/k8s-scaledown.json.env

```diff
@@ -0,0 +1,4 @@
+ACSE_POSTDEPLOY=examples/azure-cni/k8s-scale.sh
+
```

Comment:
Based on the error I saw. it makes me think we might need to sleep a minute or 2 before scaling down.